### PR TITLE
Update azure-stack-solution-template-kubernetes-dashboard.md

### PR DESCRIPTION
**Pull request by jadarsie**
Troubleshooting on custom VNET scenarios.

@mattbriggs I am pretty sure style and wording are not standard, could you please review and push fixes?

### CODE REVIEW

**Comment by PRMerger18** [comment]
@jadarsie : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: azure-stack/user/azure-stack-solution-template-kubernetes-dashboard.md [hunk 1]
```diff
@@ -122,6 +122,14 @@ You can use the dashboard. For more information on the Kubernetes dashboard, see
 
 ![Azure Stack Hub Kubernetes Dashboard](media/azure-stack-solution-template-kubernetes-dashboard/azure-stack-kub-dashboard.png)
 
+## Troubleshooting
+
+## Custom Virtual Networks
+
+If you face connectivity issues accessing the Kubernetes dashboard after you deploy Kubernetes to a [custom virtual network](https://docs.microsoft.com/en-us/azure-stack/user/kubernetes-aks-engine-custom-vnet), ensure that the target subnet/s are linked to the route table and network security group resources created by AKS engine.
```
Comment (mattbriggs) [inline azure-stack/user/azure-stack-solution-template-kubernetes-dashboard.md]: Can you change the URL (remove the locale): 

https://docs.microsoft.com/en-us/azure-stack/user/kubernetes-aks-engine-custom-vnet
to:
https://docs.microsoft.com/azure-stack/user/kubernetes-aks-engine-custom-vnet

**Comment by mattbriggs** [comment]
@jadarsie Hi ... small change before the PR can be merged. Thanks! Matt

**Comment by PRMerger18** [comment]
@jadarsie : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by mattbriggs** [comment]
#sign-off

**Comment by PRMerger20** [comment]
@ktoliver : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Pull request merged**